sqlitecluster/SQLiteNode.cpp

```diff
@@ -1082,8 +1086,9 @@ void SQLiteNode::_onMESSAGE(Peer* peer, const SData& message) {
         if (peer->params["Permafollower"] != "true" && !message.calc("Priority")) {
             STHROW("you're *not* supposed to be a 0-priority permafollower");
         }
-        // It's an error to have to peers configured with the same priority, except 0.
-        SASSERT(!_priority || message.calc("Priority") != _priority);
+
+        // It's an error to have to peers configured with the same priority, except 0.and -1
+        SASSERT(!_priority || _priority<=0 || message.calc("Priority") != _priority);
```

Comment:
`!_priority` will return for `= 0` so you your second conditional could just be `<` , also could use spaces between your operators in `_priority<=0`